keymaster/ta/operations.c

```diff
@@ -88,6 +87,7 @@ keymaster_error_t TA_abort_operation(
 				TEE_Free(operations[i].last_block.data);
 			operations[i].last_block.data = NULL;
 			operations[i].last_block.data_length = 0;
+			TEE_MemFill(operations[i].key_id, 0, sizeof(operations[i].key_id));
```

Comment:
We usually try to keep the 80 character limit (however this is not optee code itself).

Reply:
ok, will fix. I have tried to do so, looks like I have missed this case